packages/core/entity/FileDetail/index.ts

```diff
@@ -186,7 +186,7 @@ export default class FileDetail {
         return this.fileDetail.annotations.find((annotation) => annotation.name === annotationName);
     }
 
-    public async getPathToThumbnail(): Promise<string | undefined> {
+    public async getPathToThumbnail(targetSize: number | undefined): Promise<string | undefined> {
```

Comment:
Perhaps default this to undefined?
```suggestion
    public async getPathToThumbnail(targetSize?: number): Promise<string | undefined> {
```